pocs/utils/logger.py

```diff
@@ -18,14 +45,18 @@ def get_root_logger(profile='panoptes', log_config=None):
     # Get log info from config
     log_config = log_config if log_config else load_config('log').get('logger', {})
 
+    log_dir = '{}/logs'.format(os.getenv('PANDIR', '/var/panoptes/'))
+    log_fname = 'panoptes-{}-{}.log'.format(os.getpid(), datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
```

Comment:
Instead of panoptes, please use the name of the script. We don't need panoptes in the name, as the file is already in /var/panoptes.

import sys    
import os    
the_invoked_script =  os.path.basename(sys.argv[0])